### ISSUE TITLE
[Feature Request] Battery Life % for attached bluetooth devices displayed prominently in Gamescope

### ISSUE THREAD

**Issue by HunterKing** [ValveSoftware/SteamOS#929]
In KDE Plasma attached bluetooth controllers' remaining battery life can be found in the dock. However, in Gamescope, there is no clear indication of the remaining battery life left in a controller. A prominent place to find this would go a long way for usability as a home console.

**Comment by Jim-Wright** [comment]
Yes please!

**Comment by AaronBeaudoin** [comment]
The lack of this feature has been a challenge for me when a controller connected to my Steam Deck stops working and I have to second guess whether there is an issue or if the battery simply ran out. I have no idea when to charge my controllers.

**Comment by Hellmark** [comment]
Yup, definitely something that should be added, especially for those of us that use docks. I mean, why sell a dock that would require the use of controllers, and then not give info on those controllers? I recently had my controller die on me with no notification or anything mid game. Not fun.
